nameable.rb

```diff
@@ -0,0 +1,11 @@
+class Nameable
+  def initialize(nameable)
+    @nameable = nameable
+  end
+
+  attr_accessor :nameable
```

Comment:
- [ ] I don't see the `Decorator` class which must inherit from the `Namable` class. 🤔  kindly add this and make sure that:
  - It inherits from Nameable.
  - In the constructor, a namable object from params is assigned to an instance variable.
  - You have implemented the correct_name method that returns the result of the correct_name method of the @nameable.

- The `Decorator` class, might look similar to:

```ruby
class Decorator < Nameable
    attr_accessor :nameable
  
    def initialize(nameable)
        ....
    end
  

    def correct_name
           ....
    end
end
```